rpcs3/Emu/RSX/Common/texture_cache.h

```diff
@@ -1,7 +1,8 @@
-﻿#pragma once
+#pragma once
 
 #include "../rsx_cache.h"
 #include "../rsx_utils.h"
+#include "../RSXThread.h"
```

Comment:
This implies a refactor is needed. This dependency should not exist. Likely means some enum needs to be moved.

Reply:
Discord discussion result:

`rsx::get_current_renderer()->ROP_sync_timestamp` is causing the dependency and that can't be removed without refactoring which doesn't belong to this pull-request. Marking this as resolved.